modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/SchemaResourceManager.java

```diff
@@ -225,6 +237,31 @@ private JSONObject buildSystemUserSchema(List<Attribute> systemUserSchemaList) t
         }
     }
 
+    /**
+     * Builds a JSON object containing agent user schema attribute information.
+     *
+     * @param agentUserSchemaList Attribute list of SCIM agent user schema
+     * @return JSON object of agent user schema
+     * @throws CharonException
```

Comment:
```suggestion
     * @param agentUserSchemaList Attribute list of SCIM agent user schema.
     * @return JSON object of agent user schema.
     * @throws CharonException
```
complete the CharonException cause